portfolio/src/main/webapp/script.js

```diff
@@ -42,12 +42,16 @@ function showSlide(n) {
   slides[slideIndex].style.display = "block";
 }
 
-async function getComments() {
+async function getComments(num=-1) {
   const response = await fetch('/data');
   const comments = await response.json();
   const commentList = document.getElementById('data-servlet');
   commentList.innerHTML = "";
-  for(let i = 0; i<comments.length; i++) {
+  let parsedNum = parseInt(num);
+  if (parsedNum === -1 || comments.length<parsedNum) {
```

Comment:
Space before and after < for better readability. Here and everywhere.
```suggestion
  if (parsedNum === -1 || comments.length < parsedNum) {
```